### DISCUSSION TITLE
Release schedule/plans

### DISCUSSION THREAD

**Discussion by cygnus9** [ecederstrand/exchangelib#1077] [General]
Hi Erik,

I was wondering if you have a release schedule or, if not, when you're planning to make a new release. There are some unreleased changes that I'd like to use without having to depend on a particular commit or anything.

Thanks

**Comment by ecederstrand** [comment]
Sorry for the late reply. I've just published v4.7.3.

**Reply by cygnus9** [reply]
Thank you.
